LoadJSONData/LoadJSONData/Sample/Source/ItemSourceProvider.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Syncfusion.Maui.DataForm;
+
+namespace LoadJSONData
+{
+    public class ItemSourceProvider : IDataFormSourceProvider
+    {
+        public object GetSource(string sourceName)
+        {
+            if (sourceName == "City")
+            {
+                List<string> city = new List<string>() { "Chennai", "California", "Calabria", "India", "Qatar" };
+                return city;
```

Comment:
California", "Calabria", "India", "Qatar - Are countries. Update it

Reply:
Updated the countroies list.